### PR TITLE
0.14.x to develop

### PR DESCRIPTION
**Pull request by benjaoming**
Fixing conflicts in distributed.urls and kalitectl

@rtibbles student_progress.js was deleted in develop, it remains so even though it has been changed in 0.14.x

### CODE REVIEW

**Comment by benjaoming** [comment]
Only timeouts in the tests.

File: kalite/distributed/urls.py [hunk 1]
```diff
@@ -42,6 +44,9 @@
 # serving it otherwise. Cherrypy is currently serving it through modifications
 # in kalite/django_cherrypy_wsgiserver/cherrypyserver.py
 urlpatterns += patterns('',
+    url(r'^%skhan/(?P<path>.*)$' % settings.CONTENT_URL[1:], 'django.views.static.serve', {
```
Comment (MCGallaspy) [inline kalite/distributed/urls.py]: was this the merge conflict you mentioned?

Comment (benjaoming) [inline kalite/distributed/urls.py]: No I didn't change any of that, this part came straight from 0.14.x and looks good.

Comment (benjaoming) [inline kalite/distributed/urls.py]: Try merging 0.14.x into develop locally and see the conflicts. You can abort merging with `git reset --hard HEAD`

File: kalite/coachreports/static/js/coachreports/coach_reports/views.js [hunk 2]
```diff
@@ -30,6 +30,7 @@ var CoachSummaryView = BaseView.extend({
         _.bindAll(this, "set_data_model", "render");
         this.listenTo(this.model, "change:facility", this.set_data_model);
         this.listenTo(this.model, "change:group", this.set_data_model);
+        this.set_data_model();
```
Comment (MCGallaspy) [inline kalite/coachreports/static/js/coachreports/coach_reports/views.js]: @benjaoming I got a merge conflict in here as well, perhaps @rtibbles can weigh in real quick?

**Comment by MCGallaspy** [comment]
I also got this in the merge conflict, @rtibbles:
```
CONFLICT (modify/delete): kalite/coachreports/static/js/coachreports/student_progress.js deleted in HEAD and modified in
 0.14.x. Version 0.14.x of kalite/coachreports/static/js/coachreports/student_progress.js left in tree.
```

File: kalite/coachreports/static/js/coachreports/coach_reports/views.js (hunk 2 above)
Comment (rtibbles) [inline kalite/coachreports/static/js/coachreports/coach_reports/views.js]: This looks fine to me.

File: kalite/topic_tools/content_recommendation.py [hunk 3]
```diff
@@ -11,16 +11,14 @@
 
 from django.db.models import Count
 
-from kalite.topic_tools import * 
-from kalite.main.models import ExerciseLog, VideoLog, ContentLog
+# TODO(benjaoming) this wildcard is crazy, needs to be burned
```
Comment (rtibbles) [inline kalite/topic_tools/content_recommendation.py]: Och, yes.

**Comment by rtibbles** [comment]
Yep, @MCGallaspy - will reinstate that line after merging this.

**Pull request merged**